bot/core/events/registry.py

```
@@ -0,0 +1,467 @@
+import inspect
+import importlib
+import uuid
+from abc import ABC, abstractmethod
+from dataclasses import dataclass, field
+from datetime import datetime
+from typing import ClassVar, Generic
+
+from bot.utils.time import get_utc_now
+
+
+class EventType(Enum):
```

Ruff (0.8.2)

Redefinition of unused `EventType` from line 9

(F811)